### PR TITLE
Pan tilt constant vfilter

### PR DESCRIPTION
**Pull request by LRahlff**
i didnt found anything inside of pyside for getting the joystick inputs directly,
but pyjoystick made a good job

### CODE REVIEW

**Comment by Doralitze** [comment]
Could you please rebase your changes against tracker integration? Most of the changes are actually based on that one and not on main, resulting in thousands of changes compared to a few that might actually have happened.

File: requirements.txt [hunk 1]
```diff
@@ -15,4 +15,5 @@ qimage2ndarray>=1.10
 aiortc>=1.5
 qasync>=0.26
 onnxruntime-gpu>=1.16.1
-onnxruntime-openvino
\ No newline at end of file
+onnxruntime-openvino
+pyjoystick
```
Comment (Doralitze) [inline requirements.txt:19]: We need to make sure to include this dependency with the contributors and license file.

File: src/main.py [hunk 2]
```diff
@@ -18,6 +19,7 @@ def main():
     cli_server = RemoteCLIServer()
     app = QtWidgets.QApplication([])
     app.setStyleSheet(Style.APP)
+    JoystickHandler()
```
Comment (Doralitze) [inline src/main.py:22]: Why do we not store the reference here? Is it self-registering?

File: src/view/show_mode/editor/show_ui_widgets/pan_tilt_constant_show_ui.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+
+    def copy(self, new_parent: "UIPage") -> "UIWidget":
+        w = PanTiltConstantControlUIWidget(self.filter_id, self.parent, None, self.configuration)
+        super().copy_base(w)
+        return w
+
+
+    def get_config_dialog_widget(self, parent: QWidget) -> QWidget:
+        # Todo: Do we need this?
```
Comment (Doralitze) [inline src/view/show_mode/editor/show_ui_widgets/pan_tilt_constant_show_ui.py:59]: That is the widget that is being displayed once one clicks on the configuration button.

**Review by Doralitze — APPROVED** [review]
While it looks good to me, please review the notes I left.

File: src/main.py (hunk 2 above)
Comment (LRahlff) [reply src/main.py:41]: because its a static class, only exists once, but i thought it should get initialized here

**Review by Doralitze — APPROVED** [review]
LGTM

**Pull request merged**